src/PKSim.Infrastructure/ORM/Repositories/IndividualParametersSameFormulaOrValueForAllSpeciesRepository.cs

```diff
@@ -41,8 +51,8 @@ protected override void DoStart()
                IsSameFormula = isSameFormula
             };
             _individualParametersSameFormulaOrValue.Add(individualParameterSameFormulaOrValueForAllSpecies);
-            _allByContainerIdAndParameterName.Add((containerId, parameterName),
-               individualParameterSameFormulaOrValueForAllSpecies);
+            _allByContainerIdAndParameterName.Add((containerId, parameterName), individualParameterSameFormulaOrValueForAllSpecies);
+            _allByParameterPath.Add(new[] {containerPath, parameterName}.ToPathString(), individualParameterSameFormulaOrValueForAllSpecies);
```

Comment:
I need to be able to query for a PARAMETER instance where I don't have the meta data anymore.